packages/redux/src/checkout/draftOrders/types/draftOrdersAction.types.ts

```diff
@@ -0,0 +1,52 @@
+import * as actionTypes from '../actionTypes.js';
+
+/**
+ * These actions are to be used to create draft order reducer
+ */
+export const CREATE_DRAFT_ORDER_ACTIONS = [
```

Comment:
These exports could live inside the actionTypes.ts file as they only aggregate those action types.
However, there should be types representing the actions that will be dispatched through the factories. Check for example this file `packages/redux/src/addresses/types/actions.types.ts`.

Reply:
As we discussed, if I put all the types in `actionTypes.ts`, `createReducer` will break. It expects a string record and not arrays as in `draftOrdersAction.types.ts`. I followed the pattern you created at https://github.com/Farfetch/blackout/blob/main/packages/redux/src/raffles/types/rafflesActions.types.ts.

Regarding `actions.types.ts` I didn't really pay attention to this detail, I'll follow with the implementation